previews/primer/alpha/octicon_symbols_preview/default.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= render(Primer::Alpha::OcticonSymbols.new(icons: [{ symbol: :container, size: :medium }])) %>
+
+<svg xmlns="http://www.w3.org/2000/svg" height="40" width="40" aria-label="Circled container icon" tabindex="0">
+  <circle cx="20" cy="20" r="18" stroke="green" stroke-width="3px" fill="white" />
+  <use href="#octicon_container_24" x="8" y="8" />
```

Comment:
Instead of this you can have `<%= render Primer::Octicon.new(icon: :container, use_symbol: true) %>`

I'd also recommend adding multiple of the same icon to show the impact of having the symbol sheet.

Reply:
Ah yeah that's a good call! Although... is it possible to add the green circle around the container icon? I'm trying to show that the "host" SVG can contain octicons _and_ normal SVG stuff like `<circle>`, etc.